pages/learn/develop/local-mode.md

```diff
@@ -223,14 +223,15 @@ Where `dockerhub-secret.yml` is a YAML file containing my private registry usern
 Sample secrets YAML file:
 
 ```yaml
-'https://index.docker.io/v2/':
+"https://index.docker.io/v2/":
```

Comment:
I can see now those changes might be caused by an auto markdown linter running on your IDE. Can you revert these changes. Thanks!